### PR TITLE
Case key == email

### PR DESCRIPTION
**Pull request by mochizukikotaro**

### CODE REVIEW

File: lib/seed_builder/valid/string.rb [hunk 1]
```diff
@@ -31,6 +34,10 @@ def generate
 
       private
 
+      def email_str
+        "example@example.com"
```
Comment (zaru) [inline lib/seed_builder/valid/string.rb]: Good idea! 😀
I want to return a random e-mail in the future.

File: lib/seed_builder/valid/string.rb [hunk 2]
```diff
@@ -9,6 +9,9 @@ def initialize model_object:, key:
       def generate
         validator_names = validators.map{|m| m.class.name.demodulize}
 
+        # email の場合はバリデーションを無視して問答無用でメールアドレス文字列を返す
```
Comment (zaru) [inline lib/seed_builder/valid/string.rb]: please english👯

Comment (mochizukikotaro) [reply lib/seed_builder/valid/string.rb]: @zaru Oh my gosh! Sorry! I will fix.

File: lib/seed_builder/valid/string.rb (hunk 1 above)
Comment (mochizukikotaro) [reply lib/seed_builder/valid/string.rb]: @zaru AAAA. Yes! 

I will. In most cases, emails must be unique.

File: lib/seed_builder/valid/string.rb (hunk 2 above)
Comment (mochizukikotaro) [reply lib/seed_builder/valid/string.rb]: @zaru Fixed https://github.com/zaru/seed_builder/pull/33/commits/52ad9170ba67cc6318cad26c8cb70b087cc823c4

File: lib/seed_builder/valid/string.rb (hunk 1 above)
Comment (mochizukikotaro) [reply lib/seed_builder/valid/string.rb]: @zaru It's better? （I like this one.
https://github.com/zaru/seed_builder/pull/33/commits/470723859bafa3806bc1c4df55ebded63015ed1c

**Comment by zaru** [comment]
thank, i'll merge.

**Pull request merged**